tests/functional/Computer.php

```diff
@@ -77,6 +77,8 @@ public function testUpdate()
         global $CFG_GLPI;
         $saveconf = $CFG_GLPI;
```

Comment:
With your changes, saving the config and restoring it at the end of tet is now useless. You should clean the related code.